server/views/pages/recommendations/selectIndexOffence.njk

```diff
@@ -32,166 +32,188 @@
                 }) }}
             {% endif %}
 
-            <div class="govuk-grid-row">
-                <div class="govuk-grid-column-one-half">
-
-                    <h3 class='govuk-heading-m'>NOMIS</h3>
-
-                    <p class='govuk-body govuk-body-underlined'>Includes court cases, sentences and offences</p>
-
-                    <form novalidate method="post">
-                        <input type="hidden" name="_csrf" value="{{ csrfToken }}" />
-
-                        <div class="govuk-radios" data-module="govuk-radios">
-                        {% for sentence in sentences %}
-                            {% if sentence.courtDescription != courtDescription or sentence.sentenceDate != sentenceDate %}
-                                <h3 class='govuk-heading-s govuk-!-margin-bottom-1 govuk-!-margin-top-6'>Court case: {{ sentence.courtDescription }}</h3>
-                                <p class='govuk-body'>Date of sentence: {{ formatDateTimeFromIsoString({ isoDate: sentence.sentenceDate, dateOnly: true }) }}</p>
-                                {% set courtDescription = sentence.courtDescription %}
-                                {% set sentenceDate = sentence.sentenceDate %}
-                            {% endif %}
-
-                            {% for offence in sentence.offences %}
-                                <div class="govuk-radios__item">
-                                    <input class="govuk-radios__input" id="indexOffence" name="indexOffence" type="radio" value="{{ offence.offenderChargeId }}">
-                                    <label class="govuk-label govuk-radios__label" for="indexOffence">
-                                        {{ hyphenForBlank(offence.offenceDescription) }}
-                                    </label>
-
-                                    <dl class="govuk-summary-list govuk-!-margin-left-3 govuk-!-margin-bottom-4">
-                                        <div class="govuk-summary-list__row">
-                                            <dt class="govuk-summary-list__key govuk-body">
-                                                Sentence type
-                                            </dt>
-                                            <dd class="govuk-summary-list__value">
-                                                {{ hyphenForBlank(sentence.sentenceTypeDescription) }}
-                                            </dd>
-                                        </div>
+            <div class="govuk-grid-column-one-half">
+
+                <h3 class='govuk-heading-m'>NOMIS</h3>
+
+                <p class='govuk-body'>includes court cases, sentences and offences</p>
```

Comment:
Remove the HR and include this class govuk-body-underlined

See line 40 that you have removed.

Reply:
corrected